Draft new ToS (#12)
mirukucafe/legal: terms_of_service.md

# Terms of Service

Last updated: 2023-03-04

1. Introduction

> Welcome to miruku.cafe (“Company”, “we”, “our”, “us”)!

Author: Is "company" the right word to use? We are not a registered business

Reviewer: ```suggestion
Welcome to miruku.cafe (“Brand”, “we”, “our”, “us”)!
```

These Terms of Service (“Terms”, “Terms of Service”) govern your use of our website located at miruku.cafe (together or individually “Service”) operated by miruku.cafe.

Our Privacy Policy also governs your use of our Service and explains how we collect, safeguard and disclose information that results from your use of our web pages.

Your agreement with us includes these Terms and our Privacy Policy (“Agreements”). You acknowledge that you have read and understood Agreements, and agree to be bound of them.

If you do not agree with (or cannot comply with) Agreements, then you may not use the Service, but please let us know by emailing at legal@miruku.cafe so we can try to find a solution. These Terms apply to all visitors, users and others who wish to access or use Service.

2. Communications

> By using our Service, you agree to subscribe to newsletters, marketing or promotional materials and other information we may send. However, you may opt out of receiving any, or all, of these communications from us by following the unsubscribe link or by emailing at legal@miruku.cafe.

Author: Change this email to support@miruku.cafe

> By using our Service, you agree to subscribe to newsletters, marketing or promotional materials and other information we may send. However, you may opt out of receiving any, or all, of these communications from us by following the unsubscribe link or by emailing at legal@miruku.cafe.

Reviewer: ```suggestion
By using our Service, you agree to subscribe to newsletters, marketing or promotional materials and other information we may send. However, you may opt out of receiving any, or all, of these communications from us by following the unsubscribe link or by emailing at support@miruku.cafe.
```

You retain any and all of your rights to any Content you submit, post or display on or through Service and you are responsible for protecting those rights. We take no responsibility and assume no liability for Content you or any third party posts on or through Service. However, by posting Content using Service you grant us the right and license to use, modify, publicly perform, publicly display, reproduce, and distribute such Content on and through Service. You agree that this license includes the right for us to make your Content available to other users of Service, who may also use your Content subject to these Terms.

miruku.cafe has the right but not the obligation to monitor and edit all Content provided by users.

> In addition, Content found on or through this Service are the property of miruku.cafe or used with permission. You may not distribute, modify, transmit, reuse, download, repost, copy, or use said Content, whether in whole or in part, for commercial purposes or for personal gain, without express advance written permission from us.

Author: Except content under foss licenses

0.2. Use any robot, spider, or other automatic device, process, or means to access Service for any purpose, including monitoring or copying any of the material on Service.

0.3. Use any manual process to monitor or copy any of the material on Service or for any other unauthorized purpose without our prior written consent.

0.4. Use any device, software, or routine that interferes with the proper working of Service.

0.5. Introduce any viruses, trojan horses, worms, logic bombs, or other material which is malicious or technologically harmful.

0.6. Attempt to gain unauthorized access to, interfere with, damage, or disrupt any parts of Service, the server on which Service is stored, or any server, computer, or database connected to Service.

0.7. Attack Service via a denial-of-service attack or a distributed denial-of-service attack.

0.8. Take any action that may damage or falsify Company rating.

0.9. Otherwise attempt to interfere with the proper working of Service.

6. Analytics

> We may use third-party Service Providers to monitor and analyze the use of our Service.

Author: We use cloudflare but don't collect much data

> 7. No Use By Minors

Author: should we have this clause?

> 7. No Use By Minors
> 
> Service is intended only for access and use by individuals at least eighteen (18) years old. By accessing or using Service, you warrant and represent that you are at least eighteen (18) years of age and with the full authority, right, and capacity to enter into this agreement and abide by all of the terms and conditions of Terms. If you are not at least eighteen (18) years old, you are prohibited from both the access and usage of Service.

Reviewer: ```suggestion

```
Remove underage gatekeeping clause

Reviewer 2: Refer to [this reply](https://github.com/mirukucafe/legal/pull/12#discussion_r1154117660)

8. Accounts

> When you create an account with us, you guarantee that you are above the age of 18, and that the information you provide us is accurate, complete, and current at all times. Inaccurate, incomplete, or obsolete information may result in the immediate termination of your account on Service.

Author: We should definitely not have this clause

> 8. Accounts
> 
> When you create an account with us, you guarantee that you are above the age of 18, and that the information you provide us is accurate, complete, and current at all times. Inaccurate, incomplete, or obsolete information may result in the immediate termination of your account on Service.

Reviewer: ```suggestion
8. Accounts

```
remove underage gatekeeping clause

Reviewer 2: If you are going to insist on that, the way to do it would be to set the limit at 13 and ban all nsfw material and conversation, including from remote instances. You can't just remove it as if it was here for nothing.

You are responsible for maintaining the confidentiality of your account and password, including but not limited to the restriction of access to your computer and/or account. You agree to accept responsibility for any and all activities or actions that occur under your account and/or password, whether your password is with our Service or a third-party service. You must notify us immediately upon becoming aware of any breach of security or unauthorized use of your account.

You may not use as a username the name of another person or entity or that is not lawfully available for use, a name or trademark that is subject to any rights of another person or entity other than you, without appropriate authorization. You may not use as a username any name that is offensive, vulgar or obscene.

We reserve the right to refuse service, terminate accounts, remove or edit content, or cancel orders in our sole discretion.

9. Intellectual Property

> Service and its original content (excluding Content provided by users), features and functionality are and will remain the exclusive property of miruku.cafe and its licensors. Service is protected by copyright, trademark, and other laws of and foreign countries. Our trademarks may not be used in connection with any product or service without the prior written consent of miruku.cafe.

Author: Except for content under foss licenses

10. Copyright Policy

We respect the intellectual property rights of others. It is our policy to respond to any claim that Content posted on Service infringes on the copyright or other intellectual property rights (“Infringement”) of any person or entity.

> If you are a copyright owner, or authorized on behalf of one, and you believe that the copyrighted work has been copied in a way that constitutes copyright infringement, please submit your claim via email to legal@miruku.cafe, with the subject line: “Copyright Infringement” and include in your claim a detailed description of the alleged Infringement as detailed below, under “DMCA Notice and Procedure for Copyright Infringement Claims”

Author: dmca@miruku.cafe

11. DMCA Notice and Procedure for Copyright Infringement Claims

You may submit a notification pursuant to the Digital Millennium Copyright Act (DMCA) by providing our Copyright Agent with the following information in writing (see 17 U.S.C 512(c)(3) for further detail):

0.1. an electronic or physical signature of the person authorized to act on behalf of the owner of the copyright’s interest;

0.2. a description of the copyrighted work that you claim has been infringed, including the URL (i.e., web page address) of the location where the copyrighted work exists or a copy of the copyrighted work;

0.3. identification of the URL or other specific location on Service where the material that you claim is infringing is located;

0.4. your address, telephone number, and email address;

0.5. a statement by you that you have a good faith belief that the disputed use is not authorized by the copyright owner, its agent, or the law;

0.6. a statement by you, made under penalty of perjury, that the above information in your notice is accurate and that you are the copyright owner or authorized to act on the copyright owner’s behalf.

> You can contact our Copyright Agent via email at legal@miruku.cafe.

Author: dmca@miruku.cafe

> You can contact our Copyright Agent via email at legal@miruku.cafe.

Reviewer: ```suggestion
You can contact our Copyright Agent via email at dmca@miruku.cafe.
```

12. Error Reporting and Feedback

> You may provide us either directly at legal@miruku.cafe or via third party sites and tools with information and feedback concerning errors, suggestions for improvements, ideas, problems, complaints, and other matters related to our Service (“Feedback”). You acknowledge and agree that: (i) you shall not retain, acquire or assert any intellectual property right or other right, title or interest in or to the Feedback; (ii) Company may have development ideas similar to the Feedback; (iii) Feedback does not contain confidential information or proprietary information from you or any third party; and (iv) Company is not under any obligation of confidentiality with respect to the Feedback. In the event the transfer of the ownership to the Feedback is not possible due to applicable mandatory laws, you grant Company and its affiliates an exclusive, transferable, irrevocable, free-of-charge, sub-licensable, unlimited and perpetual right to use (including copy, modify, create derivative works, publish, distribute and commercialize) Feedback in any manner and for any purpose.

Author: definitely not @legal and possibly using github issues?

Reviewer: ```suggestion
You may provide us either directly via Github Issues, over https://github.com/mirukucafe/nyankey or via third party sites and tools with information and feedback concerning errors, suggestions for improvements, ideas, problems, complaints, and other matters related to our Service (“Feedback”). You acknowledge and agree that: (i) you shall not retain, acquire or assert any intellectual property right or other right, title or interest in or to the Feedback; (ii) Company may have development ideas similar to the Feedback; (iii) Feedback does not contain confidential information or proprietary information from you or any third party; and (iv) Company is not under any obligation of confidentiality with respect to the Feedback. In the event the transfer of the ownership to the Feedback is not possible due to applicable mandatory laws, you grant Company and its affiliates an exclusive, transferable, irrevocable, free-of-charge, sub-licensable, unlimited and perpetual right to use (including copy, modify, create derivative works, publish, distribute and commercialize) Feedback in any manner and for any purpose.
```

16. Termination

We may terminate or suspend your account and bar access to Service immediately, without prior notice or liability, under our sole discretion, for any reason whatsoever and without limitation, including but not limited to a breach of Terms.

> If you wish to terminate your account, you may simply discontinue using Service.

Author: you may go to the settings and delete it

> If you wish to terminate your account, you may simply discontinue using Service.

Reviewer: ```suggestion
If you wish to terminate your account, you may go into your **Account Settings** and deactivate your account.
```

18. Changes To Service

We reserve the right to withdraw or amend our Service, and any service or material we provide via Service, in our sole discretion without notice. We will not be liable if for any reason all or any part of Service is unavailable at any time or for any period. From time to time, we may restrict access to some parts of Service, or the entire Service, to users, including registered users.

19. Amendments To Terms

We may amend Terms at any time by posting the amended terms on this site. It is your responsibility to review these Terms periodically.

Your continued use of the Platform following the posting of revised Terms means that you accept and agree to the changes. You are expected to check this page frequently so you are aware of any changes, as they are binding on you.

By continuing to access or use our Service after any revisions become effective, you agree to be bound by the revised terms. If you do not agree to the new terms, you are no longer authorized to use Service.

20. Waiver And Severability

> 

Author: the fuck does this mean

No waiver by Company of any term or condition set forth in Terms shall be deemed a further or continuing waiver of such term or condition or a waiver of any other term or condition, and any failure of Company to assert a right or provision under Terms shall not constitute a waiver of such right or provision.

If any provision of Terms is held by a court or other tribunal of competent jurisdiction to be invalid, illegal or unenforceable for any reason, such provision shall be eliminated or limited to the minimum extent such that the remaining provisions of Terms will continue in full force and effect.

21. Acknowledgement

BY USING SERVICE OR OTHER SERVICES PROVIDED BY US, YOU ACKNOWLEDGE THAT YOU HAVE READ THESE TERMS OF SERVICE AND AGREE TO BE BOUND BY THEM.

22. Contact Us

> Please send your feedback, comments, requests for technical support by email: legal@miruku.cafe.

Author: email

> Please send your feedback, comments, requests for technical support by email: legal@miruku.cafe.

Reviewer: ```suggestion
Please send your feedback, comments, requests for technical support by email: support@miruku.cafe, or via Discord.
```
